### PR TITLE
*Fix Proxy setting becomes unset by user*

### PR DESCRIPTION
**Pull request by developer-av**
#1108

### CODE REVIEW

File: web/templates/user/edit_web.html [hunk 1]
```diff
@@ -85,6 +85,29 @@
                                     <textarea size="20" class="vst-textinput" name="v_aliases"><?=htmlentities(trim($v_aliases, "'"))?></textarea>
                                 </td>
                             </tr>
+                            <?php if (!empty($_SESSION['PROXY_SYSTEM'])) { echo ""; ?>
```
Comment (madeITBelgium) [inline web/templates/user/edit_web.html]: @developer-av why this empty echo ""; string?

Comment (developer-av) [inline web/templates/user/edit_web.html]: This code was copied from [vesta/web/templates/admin/edit_web.html](https://github.com/serghey-rodin/vesta/blob/master/web/templates/admin/edit_web.html#L135) in the master branch

**Comment by anton-reutov** [comment]
Should be checked by Dmitry Naumov-Socolov

**Comment by naumov-socolov** [comment]
thanks @developer-av

**Pull request merged**